### PR TITLE
Add supported mdcb version on tyk cloud 

### PR DESCRIPTION
**Pull request by ashaparulkar**
### For internal users - please add a Jira DX PR ticket to the subject!
---
### Preview Link
[preview - to add](https://deploy-preview-3148--tyk-docs.netlify.app/docs/nightly/tyk-cloud/environments-deployments/hybrid-gateways)

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

File: tyk-docs/content/tyk-cloud/tyk-cloud-mdcb-supported-versions.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+weight: 2
+menu:
+  main:
+    parent: "Tyk Cloud"
+---
+
+## Introduction
+
+
+This page list down the supported MDCB version for hybrid setup
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-cloud/tyk-cloud-mdcb-supported-versions.md]:
```suggestion
This page lists the supported MDCB version for hybrid setup
```

**Review by dcs3spp — CHANGES_REQUESTED** [review]
Thanks @ashaparulkar,

PR looks good. Please request a peer review from member of MDCB team to confirm. Once we have peer review I think we can merge and release

File: tyk-docs/content/tyk-cloud/tyk-cloud-mdcb-supported-versions.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+| v3.1.1    | v3.1.1  | v1.7.7 |
+| v3.0.9    | v3.0.9  | v1.7.11 |
+| v3.0.7    | v3.0.7  | v1.7.11 |
+| v3.0.6    | v3.0.6  | v1.7.7 |
+| v3.0.4    | v3.0.4  | v1.7.7 |
+| v3.0.3    | v3.0.3  | v1.7.7 |
+| v3.0.2    | v3.0.2  | v1.7.7 |
+| v3.1.0    | v3.1.0  | v1.7.5 |
+| v3.0.1    | v3.0.1  | v1.7.5 |
+| v3.0.0    | v3.0.0  | v1.7.5-ara |
```
Comment (sredxny) [inline tyk-docs/content/tyk-cloud/tyk-cloud-mdcb-supported-versions.md]: @ashaparulkar any ide why is required `v1.7.5-ara` and not `v1.7.5`? what does this special tag contains?

Comment (ashaparulkar) [reply tyk-docs/content/tyk-cloud/tyk-cloud-mdcb-supported-versions.md]: @sredxny , I just copied the supported mdcb versions from https://github.com/TykTechnologies/tyk-cloud-release-bundles/blob/main/prod/latest/home.json , do you want me modify it in this PR from v1.7.5-ara to v1.7.5.

Comment (excieve) [reply tyk-docs/content/tyk-cloud/tyk-cloud-mdcb-supported-versions.md]: You can look up the versions available on the Cloud here: https://github.com/TykTechnologies/tyk-cloud-release-bundles/blob/main/prod/latest/home.json. All the items in the `bundles` list that have `deprecated: false` are available.

Comment (excieve) [reply tyk-docs/content/tyk-cloud/tyk-cloud-mdcb-supported-versions.md]:
```
❯ cat prod/latest/home.json | jq '.bundles[]|select(.deprecated==false)|.components'
{
  "DashboardVersion": "v5.2.0",
  "GatewayVersion": "v5.2.0",
  "MDCBVersion": "v2.4.0",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.13"
}
{
  "DashboardVersion": "v5.1.2",
  "GatewayVersion": "v5.1.2",
  "MDCBVersion": "v2.3.0",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.13"
}
{
  "DashboardVersion": "v5.1.1",
  "GatewayVersion": "v5.1.1",
  "MDCBVersion": "v2.3.0",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.13"
}
{
  "DashboardVersion": "v5.1.0",
  "GatewayVersion": "v5.1.0",
  "MDCBVersion": "v2.3.0",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.7"
}
{
  "DashboardVersion": "v5.0.5",
  "GatewayVersion": "v5.0.5",
  "MDCBVersion": "v2.2.0",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.13"
}
{
  "DashboardVersion": "v5.0.4",
  "GatewayVersion": "v5.0.4",
  "MDCBVersion": "v2.2.0",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.12"
}
{
  "DashboardVersion": "v5.0.3",
  "GatewayVersion": "v5.0.3",
  "MDCBVersion": "v2.2.0",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.7"
}
{
  "DashboardVersion": "v5.0.2",
  "GatewayVersion": "v5.0.2",
  "MDCBVersion": "v2.2.0",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.7"
}
{
  "DashboardVersion": "v5.0.1",
  "GatewayVersion": "v5.0.1",
  "MDCBVersion": "v2.1.1",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.7"
}
{
  "DashboardVersion": "v5.0.0",
  "GatewayVersion": "v5.0.0",
  "MDCBVersion": "v2.1.1",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.7"
}
{
  "DashboardVersion": "v4.3.3",
  "GatewayVersion": "v4.3.3",
  "MDCBVersion": "v2.1.0",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.7"
}
{
  "DashboardVersion": "v4.3.2",
  "GatewayVersion": "v4.3.2",
  "MDCBVersion": "v2.0.4",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.7"
}
{
  "DashboardVersion": "v4.3.1",
  "GatewayVersion": "v4.3.1",
  "MDCBVersion": "v2.0.4",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.7"
}
{
  "DashboardVersion": "v4.3.0",
  "GatewayVersion": "v4.3.0",
  "MDCBVersion": "v2.0.4",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.7"
}
{
  "DashboardVersion": "v4.2.4",
  "GatewayVersion": "v4.2.4",
  "MDCBVersion": "v2.0.3",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.7"
}
{
  "DashboardVersion": "v4.2.3",
  "GatewayVersion": "v4.2.2",
  "MDCBVersion": "v2.0.3",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.7"
}
{
  "DashboardVersion": "v4.0.10",
  "GatewayVersion": "v4.0.10",
  "MDCBVersion": "v2.0.4",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.7"
}
{
  "DashboardVersion": "v4.0.9",
  "GatewayVersion": "v4.0.9",
  "MDCBVersion": "v2.0.3",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.7"
}
{
  "DashboardVersion": "v4.0.8",
  "GatewayVersion": "v4.0.8",
  "MDCBVersion": "v2.0.3",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.2.7"
}
{
  "DashboardVersion": "v3.2.3",
  "GatewayVersion": "v3.2.3",
  "MDCBVersion": "v1.8.1",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.0.15"
}
{
  "DashboardVersion": "v3.0.9",
  "GatewayVersion": "v3.0.9",
  "MDCBVersion": "v1.7.11",
  "MservVersion": "v0.9.9-beta.4",
  "RedisVersion": "6.0.15"
} 
```

**Comment by ashaparulkar** [comment]
Thanks @excieve , @sredxny for review.
@dcs3spp , I have corrected the PR after peer review

**Review by dcs3spp — APPROVED** [review]

**Comment by dcs3spp** [comment]
@ashaparulkar Many thanks for submitting the changes.

@sredxny or @excieve would you be able to re-review and then we can merge and release.

**Review by excieve — COMMENTED** [review]
@dcs3spp It looks like something went wrong with the merges as the original version of the change appears to be back. Could you please check it?

**Comment by excieve** [comment]
> @dcs3spp It looks like something went wrong with the merges as the original version of the change appears to be back. Could you please check it?

Ignore me, it looks good to me.

**Review by excieve — APPROVED** [review]

**Review by sredxny — APPROVED** [review]
LGTM 🚀

**Pull request merged**